src/lib/scheduler/node_queue_scheduler.hpp

```diff
@@ -12,63 +12,57 @@ namespace hyrise {
 /*
  * GENERAL SCHEDULING CONCEPT
  *
- * Everything that needs to be processed is encapsulated in tasks. For example, in the context of the database
- * the OperatorTasks encapsulates database operators (here, it only encapsulates the execute function).
- * A task will then be pushed by a Scheduler into a TaskQueue and pulled out by a Worker to be processed.
+ * The scheduler is the main entry point and there are currently two alternative implementations: the
+ * ImmediateExecutionScheduler (single-threaded, for benchmarking and debugging) and the NodeQueueScheduler (multi-
+ * threaded).
+ *
+ * Everything that needs to be processed is encapsulated in tasks. A task will be pushed into a TaskQueue by a
+ * scheduler and pulled by a Worker to be processed.
```

Comment:
Shouldn't most of this (also) be in `abstract_scheduler.hpp`?

Reply:
Right, I moved the general processing stuff there while the NQS-specific paragraphs remain in the NQS header.